### PR TITLE
Add png resolution option to saveGIF and ani.option function

### PR DESCRIPTION
**Pull request by yulijia**
Add png resolution option to saveGIF and ani.option function, by default it should be 72 dpi.

### CODE REVIEW

**Comment by yulijia** [comment]
Thanks. I need write a if statement, the resolution option only support png, jpeg, bmp and tiff device.

File: R/saveGIF.R [hunk 1]
```diff
@@ -83,9 +83,17 @@ saveGIF = function(
   ani.dev = ani.options('ani.dev')
   if (is.character(ani.dev)) ani.dev = get(ani.dev)
   img.fmt = paste(img.name, '%d.', file.ext, sep = '')
-  if ((use.dev <- ani.options('use.dev')))
-    ani.dev(file.path(tempdir(), img.fmt), width = ani.options('ani.width'),
-            height = ani.options('ani.height'))
+
+  if ((use.dev <- ani.options('use.dev'))){
+    if (sum(grepl(ani.options('ani.dev'),c("png","bmp","jpeg","tiff")))){
```
Comment (yihui) [inline R/saveGIF.R]: I'd use `any()` instead of `sum()`. Although `sum()` works, it feels hackish.

Spaces after commas.

File: R/saveGIF.R [hunk 2]
```diff
@@ -83,9 +83,17 @@ saveGIF = function(
...
+      ani.dev(file.path(tempdir(), img.fmt), width = ani.options('ani.width'),
+              height = ani.options('ani.height'), res = ani.options('ani.res'))
+      # ,bg = ani.options('ani.bg')
+    }else{
```
Comment (yihui) [inline R/saveGIF.R]: spaces

File: NEWS [hunk 3]
```diff
@@ -6,6 +6,9 @@ MAJOR CHANGES
   CMD check on CRAN for unknown reasons; all examples are still available on
   Github https://github.com/yihui/animation/tree/master/inst/examples and they
   will be moved to https://yihui.name/animation in the future
+  
+  o add a new option ani.res() to ani.options(), to control the resolution of png, 
+  jepg, bmp and tiff images when using saveGIF() (#99).
```
Comment (yihui) [inline NEWS]: 2.5 has been released. This entry belongs to 2.6.

File: NEWS [hunk 4]
```diff
@@ -6,7 +13,7 @@ MAJOR CHANGES
   CMD check on CRAN for unknown reasons; all examples are still available on
   Github https://github.com/yihui/animation/tree/master/inst/examples and they
   will be moved to https://yihui.name/animation in the future
-
+
```
Comment (yihui) [inline NEWS]: extra spaces introduced here

File: NEWS [hunk 5]
```diff
@@ -1,3 +1,10 @@
+      CHANGES IN animation VERSION 2.6
```
Comment (yihui) [inline NEWS]: could you indent the title to the same level as other titles in this file?

**Pull request merged**